script.js

```diff
@@ -18,21 +25,59 @@ if (data && data.length > 0) {
 }
 
 function AddTodo(todoText) {
-    const element = document.createElement("LI")
-    const elementText = document.createTextNode(todoText)
-    element.appendChild(elementText)
+        const element = document.createElement("LI");
+	const elementBtn = document.createElement("button");
+	const elementText = document.createTextNode(todoText);
+	var elementId = nameId+loop;
+	loop++;
+	elementBtn.id = elementId;
```

Comment:
you could wrap the creation of the button in an individual function which you use here, so the code is more readable

Reply:
f.e. createButton(ID)